CONTRIBUTING.md

```diff
@@ -30,6 +30,18 @@ These steps outline the general contribution workflow:
 * Make sure the tests pass, and add any new tests as appropriate.
 * Submit a pull request to the original repository. (see [Format of Pull Requests](#format-of-pull-requests))
 
+## Required tools
+To allow scripts and `make` targets working correctly, ensure you have following tools installed in your system:
+
+* golang
```

Comment:
1.14+ I believe

Nit:
```suggestion
* go v1.14+
```

Reply:
This is a moving target hence I didn't put version number in it. We require specific version number in `go.mod`

Reply:
We can always update if need specific versions later, but it won't work pre 1.14 no?

Reply:
This is exactly what locking in `go.mod` is for. No need to add documentation entry which can easily go stale.